spec/features/a_user_can_subscribe_to_the_place_feed.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'A user subscribe to the place Atom feed' do
+  scenario 'and consult the complete correct feed page' do
+    CsvImportService.new('Place', Rails.root.join('spec/fixtures/csv/valid.csv'), state: 'active').save
```

Comment:
We shouldn't load fixtures in specs, but use FactoryGirl to create objects needed for the tests.

See https://github.com/mainelabs/techlist/blob/master/spec/features/admin/an_admin_can_accept_a_new_place_spec.rb#L5, for instance.